api/HTMLElement.json

```diff
@@ -2227,10 +2227,10 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLElement/oncopy",
           "support": {
             "chrome": {
-              "version_added": "71"
+              "version_added": "1"
```

Comment:
The original data (Chrome 71) came from https://github.com/mdn/browser-compat-data/pull/3502 which in turn used Confluence:
http://web-confluence.appspot.com/#!/catalog?releases=%5B%22Chrome_70.0.3538.67_OSX_10.13.6%22,%22Chrome_70.0.3538.67_Windows_10.0%22,%22Chrome_71.0.3578.80_OSX_10.14.0%22,%22Chrome_71.0.3578.80_Windows_10.0%22%5D&q=%22oncopy%22

What happened was that `oncopy` was moved from `Element` to `HTMLElement` here:
https://chromium.googlesource.com/chromium/src/+/bce553b6022a43cc4f9744de69ef85a753f47624

This is a similar situation to https://github.com/mdn/browser-compat-data/issues/6545, and I'd like to handle it as suggested in https://github.com/mdn/browser-compat-data/issues/6545#issuecomment-679147588, by treating it as supported on `HTMLElement` with no note even when that support was inherited from `Element`.

Reply:
To consolidate the size of this PR, I cherry-picked these changes as #7242.

Reply:
Thanks!